presto-atop/src/test/java/io/prestosql/plugin/atop/LocalAtopQueryRunner.java

```diff
@@ -22,6 +22,7 @@
 import java.util.TimeZone;
 
 import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.prestosql.testing.LocalQueryRunnerBuilder.createLocalQueryRunner;
```

Comment:
Typically a builder is an inner class. So `LocalQueryRunner.Builder` instead of `LocalQueryRunnerBuilder`.